### PR TITLE
Make 64-bit control field namespaces

### PR DESCRIPTION
**Pull request by connojd**

### CODE REVIEW

**Comment by rianquinn** [comment]
this one has issues with coveralls

**Comment by connojd** [comment]
@rianquinn it should be ready now

File: bfvmm/src/vmcs/test/test.cpp [hunk 1]
```diff
@@ -118,6 +118,95 @@ enable_entry_ctl(uint64_t control)
     vmcs::vm_entry_controls::set(ctls | control);
 }
 
+
+void
+proc_ctl_allow1(uint64_t mask)
+{ g_msrs[msrs::ia32_vmx_true_procbased_ctls::addr] |= mask << 32; }
+
+//void
```
Comment (rianquinn) [inline bfvmm/src/vmcs/test/test.cpp]: Please remove these comments once all of this is done

Comment (connojd) [inline bfvmm/src/vmcs/test/test.cpp]: I will..those were part of the reason coveralls wasn't passing

**Pull request merged**